vehicle/autoware_raw_vehicle_cmd_converter/test/test_autoware_raw_vehicle_cmd_converter.cpp

```diff
@@ -110,11 +109,32 @@ TEST(ConverterTests, LoadValidPath)
   BrakeMap brake_map;
   SteerMap steer_map;
 
+  std::vector<double> map_row_idx = {0.0, 5.0, 10.0};
+  std::vector<double> map_column_idx = {0.0, 0.5, 1.0};
+  std::vector<std::vector<double>> map_value = {
+    {0.0, -0.3, -0.5}, {1.0, 0.5, 0.0}, {3.0, 2.0, 1.5}};
+
   // for valid path
   EXPECT_TRUE(loadAccelMapData(accel_map));
   EXPECT_TRUE(loadBrakeMapData(brake_map));
   EXPECT_TRUE(loadSteerMapData(steer_map));
 
+  // for get function in acceleration
```

Comment:
The name of this test is `LoadValidPath`, but it also checks the value. Is there a better name?

Reply:
Fixed in cfb244ec84373df4c7c0f2ccf5a28d3f9b6bfbfb